AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4007,6 +4008,44 @@ function Debug-AzStorageAccountEntraKerbAuth {
             Debug-EntraKerbAdminConsent -StorageAccountName $StorageAccountName -checkResult $checks["CheckAdminConsent"]
         }
 
+        #
+        # Check if WinHttpAutoProxySvc and iphlpsvc service is running
+        #
+        if (!$filterIsPresent -or $Filter -match "CheckWindowServices")

```

Comment:
Could you make this two separate checks? This would help get more fine grained information across at a glance. We can call them `CheckWinHttpAutoProxySvc` and `CheckIpHlpSvc`.